### PR TITLE
RDR: setting client info

### PR DESCRIPTION
**Pull request by raaizik**
Writes to the newly added fields of `ReportStatusRequest` which are then exported by means of the changes introduced in [RDR: exporting client info #2504 ](https://github.com/red-hat-storage/ocs-operator/pull/2504)

### CODE REVIEW

**Review by umangachapagain — COMMENTED** [review]
Looks good to me.

Which is the PR that creates the final ConfigMap?

File: service/status-report/main.go [hunk 1]
```diff
@@ -138,7 +145,9 @@ func main() {
 
 	status := providerclient.NewStorageClientStatus().
 		SetPlatformVersion(pltVersion).
-		SetOperatorVersion(oprVersion)
+		SetOperatorVersion(oprVersion).
+		SetClusterID(string(clusterID)).
```
Comment (Madhu-1) [inline service/status-report/main.go:149]: why are we sending the clusterID again? i believe it should be available on the storageConsumer CR. isn't that the case?

Can't we send this information when onboarding not each time during the status report?

Comment (raaizik) [reply service/status-report/main.go:149]: See [discussion](https://github.com/red-hat-storage/ocs-operator/pull/2504#discussion_r1524166253)

**Comment by raaizik** [comment]
> Looks good to me.
> 
> Which is the PR that creates the final ConfigMap?

@umangachapagain it's still a [WIP](https://github.com/red-hat-storage/ocs-operator/pull/2440)

File: service/status-report/main.go (hunk 1 above)
Comment (raaizik) [reply service/status-report/main.go:149]: > See discussions [(2)](https://github.com/red-hat-storage/ocs-operator/pull/2504#discussion_r1524179758)

Comment (Madhu-1) [reply service/status-report/main.go:149]: Resolving it as got the answer

**Pull request merged**